### ISSUE TITLE
A directory "master" identification feature.

### ISSUE THREAD

**Issue by NyaNguyen** [lordmulder/DoubleFilerScanner#1]
Hello,

Very nice clean program that gets the job done.  One option that is pretty major is the ability  to mark / check the parent directories that should not compare the any file within that each other.  For example, if we have already scanned the files in the directory, it would save time by not scanning it again and simply compare the files in that directory / sub-directory with any other directories that are not marked as a "master".   Thanks!

**Comment by lordmulder** [comment]
I'm not exactly sure I understand what you want 😕

**Comment by vbc4ejyg** [comment]
nya13, the problem with your suggestion is if a hash changes between checks, you wouldn't know it if you cache the results.  You want all hashes to be computed at the time the checker runs.

**Comment by NyaNguyen** [comment]
> 
> 
> I'm not exactly sure I understand what you want 😕

Basically, have the ability to mark one or many folders to not compare their hashes against one another and only have their hashes be compared with files from "non-master" directories.  A practical example of this is I have 1 million files to compare.  I run the program to check for duplicates.  It identifies the duplicates and I delete the duplicates and keep one of each hash group.  2 weeks later I have new files coming in and need to check if they are duplicates.  Obviously, I do not need to check the initial list of files against each other as I already did that 2 weeks ago.  Do computer the hash for all files, but do not compare the files on those folders we marked as "master".
